### PR TITLE
✨ feat: Integrate with UI Plaid.

### PR DESCRIPTION
**Pull request by joartola**

### CODE REVIEW

**Comment by joartola** [comment]
https://user-images.githubusercontent.com/31057068/221443269-14a8c46c-ebd2-4864-a671-ed49193e112c.mov

File: webapp/components/JoinDALNButton.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  onSuccess?: () => void;
+}
+export default function JoinDALNButton({
+  linkToken,
+  onClick = () => {},
+  isDisabled,
+  onSuccess = () => {},
+  ...props
+}: JoinDALNButtonProps) {
+  const sendPublicTokenToBackend = useCallback(
```
Comment (kgelpes) [inline webapp/components/JoinDALNButton.tsx:28]: We could call this using react query, probably better for handling states

File: webapp/hooks/useCreateLinkToken.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  };
+
+  useEffect(() => {
+    generateToken();
+  }, []);
+
+  return { ...state, retryGenerateToken: generateToken };
+};
+
+export default useCreateLinkToken;
```
Comment (kgelpes) [inline webapp/hooks/useCreateLinkToken.tsx]: this file could be replaced with useQuery from react-query

**Review by kgelpes — CHANGES_REQUESTED** [review]

**Review by kgelpes — APPROVED** [review]
looks good!

**Pull request merged**